research-software-directory/KIN-RPD
# Kin demo rc

# Home page setup 

Closes #2

Changes proposed in this pull request:
* Remove stats and news from homepage
* Data generation module reduced to projects, organisations and meta pages      

How to test:
* `make start` to build app and generate test data
* confirm page looks like in the example

## Example KIN RPD homepage

![image](https://github.com/research-software-directory/KIN-RPD/assets/9204081/e6355417-380b-4b06-89ca-3ad767fa571c)


PR Checklist:
